predictsignauxfaibles/__main__.py

```diff
@@ -39,8 +51,31 @@ def load_conf(args):
     return model_conf
 
 
+def load_datasets_from_conf(args_ns, conf):
```

Comment:
please provide type hints for the function args — that's really helpful when reading the code :)

Reply:
what does `ns` stand for in `args_ns` ?